models/workload.go

```diff
@@ -48,6 +49,8 @@ type WorkloadListItem struct {
 	ResourceVersion string `json:"resourceVersion"`
 
 	// Define if Workload has an explicit Istio policy annotation
+	// Istio supports this as a label as well - this will be defined if the label is set, too.
+	// If both annotation and label are set, if any is false, injection is disabled.
 	// It's mapped as a pointer to show three values nil, true, false
 	IstioInjectionAnnotation *bool `json:"istioInjectionAnnotation,omitempty"`
```

Comment:
Probably you may need to differentiate if you are using labels vs annotations and tell that to the UI which should build a patch depending of the case

https://github.com/kiali/kiali-ui/blob/master/src/components/IstioWizards/WizardActions.ts#L1407

Suggestion a new field with "IstioInjectionLabel" in the similar way, and let the UI to use one or another.

Reply:
I originally thought of that but decided against it because of:

1. the amount of change would be a lot more (it would also require a change to the operator also)
2. adding yet another config option that will likely never change and will likely never have a value different from the annotation label name

In other words, the name of this label is (and will always be) the same as the name of the annotation (`sidecar.istio.io/inject`). I don't see any benefit to creating a separate config option here because it will likely never be the case that the annotation label will be one name and the name of the label will be different than that.

Do you see a benefit to creating a separate one?